LICENSE

```diff
@@ -20,3 +20,54 @@ AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
 LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
 OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
 THE SOFTWARE.
+
+
+Furthermore, the files pytest_leaks/refleak_*.py and
+pytest_leaks/support.py are covered by the PSF license:
```

Comment:
The short name for this license is `Python-2.0` https://spdx.org/licenses/Python-2.0.html

Sticking it in here will probably make GitHub's license detection fail.  I suggest having two files, LICENSE and LICENSE-PSF.txt

And then glue them together by mentioning them both in the setup.py : `license='MIT and Python-2.0'` , and possibly mention both in the README too, and also ensure the next release includes both license files in the sdist.

Reply:
Thanks, going to postpone license splitting in separate PR

Reply:
As predicted, GitHub is now showing "View license" because it cant detect it.  Not terrible, but I think it is worth the effort to convince/trick GitHub that it is MIT license, because the addition of PSF license has zero impact on 99% of users because they are already already running CPython.  And the other 1% are distro packagers, and they should be checking the licenses in the sdist (and completely ignoring whatever GitHub says the license might be).

Reply:
This may be a silly question, but does the PSF come from the template? Because at this stage and number of authors, it would also not be an issue to simply relicense the code. Although, it is probably not really necessary.

Reply:
PSF comes because refleak_*.py are copypasted code from cpython.
Since we're not PSF ourselves, the license probably won't make sense for code we write.